luarules/configs/ffa_startpoints/README.md

```diff
@@ -0,0 +1,242 @@
+# FFA start setup gadget
+
+This directory holds FFA start points configs intended to be consumed by the
+**FFA start setup** gadget (`game_ffa_start_setup.lua`), which is responsible
+for setting up start points and start boxes in FFA and TeamFFA games.
+
+## Why
+
+Start positions in FFA and TeamFFA are an extremely important variable, as a
+huge part of the early game is trying to determine who's where and what one
+should do based off that.
+
+An annoying quirk when using start boxes is that they must be set from the lobby
+and are tied to a specific team (e.g. team 1 will use start box 1), which
+removes the uncertainty factor as all players know immediately where everybody
+starts before the game has even launched.
+
+When not using start boxes, players will be randomly placed at the start of the
+match based on start positions defined by the map's `mapinfo.lua`. However, not
+all maps properly define start positions in a way that will work for FFA games,
+and some maps do not even define enough start positions to allow for all players
+to spawn properly, even if the map could actually support it.
+
+On top of that, the start positions defined from `mapinfo.lua` are a plain list,
+which is not flexible enough to define multiple layouts of start positions
+depending on the number of contestants (e.g. one specific layout for 5-way,
+another one for 7-way).
+
+The **FFA start setup** gadget's purpose is to address all of these issues.
+
+## How
+
+If using start boxes, the gadget will shuffle them at game start so as to make
+sure that nobody can know where anybody else is based off the start boxes
+defined in the lobby.
```

Comment:
Maybe this should be an option to do shuffling of startboxes. Teams might want to set up boxes and assign them to teams in very specific way e.g. in case of unbalances teams etc.

Reply:
If we have complaints about this why not, but I've only ever heard of complaints about the fact FFA startboxes are known at lobby time, so I'm inclined to have it as default and not to have it as an option.

Reply:
Sounds fair, we might just not need it.